src/Compilers/Test/Utilities/CSharp/SemanticModelTestBase.cs

```diff
@@ -220,5 +219,41 @@ protected CompilationUtils.SemanticInfoSummary GetSemanticInfoForTest(string tes
         {
             return GetSemanticInfoForTest<ExpressionSyntax>(testSrc);
         }
+
+        protected string GetOperationTreeForTest<TSyntaxNode>(CSharpCompilation compilation)
+            where TSyntaxNode: SyntaxNode
+        {
+            var tree = compilation.SyntaxTrees[0];
+            var model = compilation.GetSemanticModel(tree);
+            SyntaxNode syntaxNode = GetSyntaxNodeOfTypeForBinding<TSyntaxNode>(GetSyntaxNodeList(tree));
+            if (syntaxNode == null)
+            {
+                return null;
+            }
+
+            var operation = model.GetOperationInternal(syntaxNode);
```

Comment:
so when testing.. do we want to explicit on IOperation we get from the API? or we use the default behavior?

can user get IOperation off a node? or is it only internal, testing stuff?